packages/scheduler/src/Scheduler.js

```diff
@@ -72,28 +72,28 @@ function scheduleHostCallbackIfNeeded() {
   }
 }
 
-function flushFirstTask() {
-  const currentlyFlushingTask = firstTask;
-
+function flushTask(task) {
```

Comment:
Realizing now that the commit message doesn't really make sense anymore. Originally I thought I would need to be able to flush a task at an arbitrary point in the list, but that turned out not to be the case. I'll keep the change for type safety reasons, though.